clean/rayne_pd_cprr.py

```diff
@@ -131,7 +133,129 @@ def clean_rank(df):
     return df
 
 
-def clean():
+def strip_leading_commas(df):
+    for col in df.columns:
+        df[col] = df[col].str.replace(r"^\'", "", regex=True)
+    return df
+
+
+def split_names14(df):
+    names = (
+        df.focus_officer.str.lower()
+        .str.strip()
+        .str.replace(r"(1\)|2\)|3\)|4\)) ", "", regex=True)
+        .str.extract(r"(sgt|ptl|pfc)?\.? ?(\w+) (\w+)")
+    )
+
+    df.loc[:, "rank_desc"] = (
+        names[0]
+        .fillna("")
+        .str.replace(r"sgt", "sergeant", regex=False)
+        .str.replace(r"ptl", "patrol", regex=False)
+    )
+    df.loc[:, "first_name"] = names[1].fillna("")
+    df.loc[:, "last_name"] = names[2].fillna("")
+    return df.drop(columns=["focus_officer"])
+
+
+def split_rows_w_multiple_allegations14(df):
+    df.loc[:, "allegation"] = (
+        df.complaint.str.lower()
+        .str.strip()
+        .str.replace(r"^(a\)|1\)) ", "", regex=True)
+        .str.replace(r"(b\)|c\)|1\)|2\)|3\))", "|", regex=True)
```

Comment:
You can write it shorter like this: `r"(b|c|1|2|3)\)"` or even shorter `r"[bc1-3]\)`.